src/features/movies/PopularMoviesPage/index.js

```diff
@@ -0,0 +1,39 @@
+import { useEffect, useState } from "react";
+import { Container } from "../../../common/Container";
+import TilesSection from "../../../common/TilesSection";
+import MovieTile from "../MovieTile";
+
+const PopularMoviesPage = () => {
+  const base_url = "https://api.themoviedb.org/3";
+  const api_key = "api_key=0e017d1cf3aa44c76887c24c592892f0";
+
+  const [popularMovies, setPopularMovies] = useState([]);
+
+  useEffect(() => {
+    fetch(`${base_url}/discover/movie?sort_by=popularity.desc&${api_key}`)
+      .then((response) => response.json())
+      .then((data) => setPopularMovies(data.results));
```

Comment:
I think we can remove unnecessary bracket

![obraz](https://user-images.githubusercontent.com/118211320/221407400-41f40c92-6acc-4908-b2c5-61ea67a5eacd.png)

Reply:
It's a feature with prettier formatter